src/rules/ban_unknown_rule_code.rs

```diff
@@ -7,6 +7,8 @@ use crate::{Program, ProgramRef};
 #[derive(Debug)]
 pub struct BanUnknownRuleCode;
 
+pub(crate) static CODE: &str = "ban-unknown-rule-code";
```

Comment:
I know `static` also works, but in other rule implementations we use `const` to declare the rule `CODE`. So for consistency, it'd be great to use `const` here too.